src/v3/test/integration/authenticator-expired-password.test.tsx

```diff
@@ -68,4 +61,108 @@ describe('authenticator-expired-password', () => {
+    await findByText(/Password requirements/);
+
+    const submitButton = await findByText('Change Password', { selector: 'button' });
+    const newPasswordEle = await findByTestId('credentials.passcode') as HTMLInputElement;
+
+    await user.type(newPasswordEle, 'superSecretP@ssword12');
+
+    await user.click(submitButton);
+
+    const confirmPasswordError = await findByTestId('confirmPassword-error');
+
+    expect(confirmPasswordError.innerHTML).toBe('This field cannot be left blank');
+    expect(authClient.options.httpRequestClient).not.toHaveBeenCalledWith(
+      'POST',
+      'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+      expect.anything(),
+    );
+  });
+
+  it('should not make network request when only confirm password has a value', async () => {
+    const {
+      authClient, user, findByTestId, findByText,
+    } = await setup({ mockResponse });
+
+    await findByText(/Your password has expired/);
+    await findByText(/Password requirements/);
+
+    const submitButton = await findByText('Change Password', { selector: 'button' });
+    const confirmPasswordEle = await findByTestId('confirmPassword') as HTMLInputElement;
+
+    await user.type(confirmPasswordEle, 'abc123');
+    await user.click(submitButton);
+
+    const newPasswordError = await findByTestId('credentials.passcode-error');
+
+    expect(newPasswordError.innerHTML).toBe('This field cannot be left blank');
+    expect(authClient.options.httpRequestClient).not.toHaveBeenCalledWith(
+      'POST',
+      'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+      expect.anything(),
+    );
+  });
+
+  it('should not make network request when fields are not matching', async () => {
+    const {
+      authClient, user, findByTestId, findByText,
+    } = await setup({ mockResponse });
+
+    await findByText(/Your password has expired/);
+    await findByText(/Password requirements/);
+
+    const submitButton = await findByText('Change Password', { selector: 'button' });
+    const newPasswordEle = await findByTestId('credentials.passcode') as HTMLInputElement;
+    const confirmPasswordEle = await findByTestId('confirmPassword') as HTMLInputElement;
+
+    const password = 'superSecretP@ssword12';
+    await user.type(newPasswordEle, password);
+
+    await user.type(confirmPasswordEle, 'abc123');
+    await user.click(submitButton);
+
+    const confirmPasswordError = await findByTestId('confirmPassword-error');
+
+    expect(confirmPasswordError.innerHTML).toBe('New passwords must match');
+    expect(authClient.options.httpRequestClient).not.toHaveBeenCalledWith(
+      'POST',
+      'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+      expect.anything(),
+    );
+  });
+
+  // TODO: validate function is not being overwritten here, but works in real org, why?
+  it.skip('should not make network request without completing any password fields', async () => {
```

Comment:
An odd issue here is happening (only in test) after rebasing with 0.0, the validate function is not being overwritten. When I manually test, it works just fine and I can see both errors, but in this test it fails because the function isnt overwritten. I debugged through it and I can see the validate function being re-set but still doesnt get executed. Still debugging, but wanted to push up the other changes for re-review in the meantime.